desk/app/reel.hoon

```diff
@@ -23,6 +23,9 @@
       civ=ship
       our-metadata=(map cord metadata:reel)
   ==
+++  url-for-token
+  |=  [vic=cord our=ship token=cord]
+  (crip "{(trip vic)}{(trip (scot %p our))}/{(trip token)}")
```

Comment:
should include `/lure` in the middle

Reply:
Just remembered that the `/lure` is supposed to be in the `vic` (see [here](https://github.com/tloncorp/lure/blob/develop/desk/app/reel.hoon#L38)) and it was only missing due to the config of the ship we tested on.